validator/client/src/main/java/tech/pegasys/teku/validator/client/restapi/apis/PostKeys.java

```diff
@@ -30,13 +32,32 @@ public PostKeys() {
             .operationId("ImportKeystores")
             .summary("Import Keystores")
             .description("Import keystores generated by the Eth2.0 deposit CLI tooling.")
+            .requestBodyType(ValidatorTypes.POST_KEYS_REQUEST)
+            .response(SC_OK, "Success response", ValidatorTypes.POST_KEYS_RESPONSE)
             // TODO add request and response
             .withAuthenticationResponses()
             .build());
   }
 
   @Override
   public void handle(final RestApiRequest request) throws JsonProcessingException {
+    //    final PostKeysRequest body = request.getRequestBody();
+    //    if (body.getKeystores() == null || body.getPasswords() == null) {
+    //      request.respondError(SC_BAD_REQUEST, "keystores or passwords list was not receieved.");
+    //      return;
+    //    }
+    //    if (body.getKeystores().isEmpty() && body.getPasswords().isEmpty()) {
+    //      request.respondOk(List.of());
+    //    } else if (body.getKeystores().size() != body.getPasswords().size()) {
+    //      request.respondError(
+    //          SC_BAD_REQUEST, "Different number of passwords and keystores supplied, cannot
+    // process.");
+    //      return;
+    //    }
+
+    // TODO call keyManager.importValidators(body.getKeystores(), body.getPasswords(),
+    // body.getSlashingProtection());
+    // and set request.respondOk if it doesnt fail  - it shouldn't fail...
```

Comment:
Also not a big deal but I'd probably remove the commented out code.

Reply:
can do, its just basically some of the error checking and was breaking a test...